### ISSUE TITLE
Development Day @ QuantEcon Labs

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/sphinxcontrib-jupyter#71]
We are getting a couple of RA's to work on closing a few of the issues next Tuesday (21-August)

One of our main users has requested the following priorities to help in preparing Jupiter notebooks for courses.

```
My priorities though would be:
1. Static files: support for directory structure and correct copying
2. Slides options in meta-data
3. :class: solution
```

These are Issues #69, #68, #58, #22

**Comment by mmcky** [comment]
`:class: solutions` has been implemented in PR #72

**Comment by mmcky** [comment]
@FelipeMaldonado is starting to work on a `jupyter` directive which would allow more control over the RST to IPYNB conversion. The initial use case will be:

```
.. jupyter:: 
   :break-cell:
```
to enable one to force a break in cells.
